_assets/js/netlify/formatDates.js

```diff
@@ -0,0 +1,6 @@
+export default function formatDates(date) {
```

Comment:
good foresight making this a module

in my experience apps usually end up with quick a few different date formats - it might be good to name this function `formatPublishDate` or something to leave room for `formatSomeOtherDate` in the future

Reply:
Good catch!